src/OrchardCore/OrchardCore.Contents.Core/Services/ContentQueryService.cs

```diff
@@ -0,0 +1,91 @@
+using System.Linq;
+using OrchardCore.ContentManagement;
+using OrchardCore.ContentManagement.Records;
+using OrchardCore.Contents.Core.Options;
+using YesSql;
+using YesSql.Services;
+
+namespace OrchardCore.Contents.Services
+{
+    public class ContentQueryService : IContentQueryService
+    {
+        private readonly YesSql.ISession _session;
+        public ContentQueryService(
+            YesSql.ISession session
+            )
+        {
+            _session = session;
+        }
+        public IQuery<ContentItem, ContentItemIndex> GetQueryByOptions(OrchardCore.Contents.Core.Options.ContentOptions options)
```

Comment:
Add new line before

Reply:
Is there another type of `ContentOptions` coz I see the full qualified name everywhere?